CONTRIBUTING.md

```diff
@@ -188,15 +188,22 @@ $ export GCLOUD_TESTS_GOLANG_KEYRING=projects/$GCLOUD_TESTS_GOLANG_PROJECT_ID/lo
 $ gsutil kms authorize -p $GCLOUD_TESTS_GOLANG_PROJECT_ID -k $GCLOUD_TESTS_GOLANG_KEYRING/cryptoKeys/key1
 
 # Create KMS Key in one region for Bigtable
+$ gcloud kms keyrings create $MY_KEYRING --location $MY_SINGLE_LOCATION
```

Comment:
As we use the same keyring for storage above, https://github.com/googleapis/google-cloud-go/pull/4356/files#L179, will this fail if you work through this contributing guide planning to contribute to both as the keyring has already been created?

Reply:
It's not the same keyring. It has a different location and shows up in the UI with a different location. Without creating the local keyring, the create-key below fails -- or at least it did for me.